src/App.vue

```diff
@@ -258,15 +262,15 @@ h2 {
 }
 
 .link {
-    @apply hover:(text-dark-300 underline underline-dark-300);
+    @apply hover:(focus:text-red-500 hover:text-red-500);
 }
 
 .link-secondary {
     @apply hover:(text-dark-400 underline underline-dark-400);
 }
 
 .dark .link {
-    @apply hover:(text-gray-300 underline underline-gray-300);
+    @apply hover:(focus:text-red-400 hover:text-red-400);
```

Comment:
Isn't `hover:` unnecessary for `text-red-x`? Since the parent is `hover:` already.
Shouldn't `hover:` and `focus:` also be separate, or am I wrong?